### ISSUE TITLE
Fails to create rows with connected models using prisma 2.0.0-beta.1

### ISSUE THREAD

**Issue by emolr** [prisma/studio#397]
1. Prisma2 version (`prisma2 -v`): 2.0.0-beta.1

2. Logs from Developer Tools Console, if any: Visible in video

3. Does the issue persist even after updating to the latest `prisma2` alpha (`npm i -g prisma2@alpha`)? Doesn't apply

4. Prisma2 schema (if relevant):
```
datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

model Post {
  id        Int     @id @default(autoincrement())
  title     String
  content   String?
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  Int?
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  posts Post[]
}

```

This issue occurs with the getting started example from the new prisma website
I made a video showing what happens : [Studio fails to add connections](https://res.cloudinary.com/histudios/video/upload/v1585720839/prisma_studio_with_beta.1_rytgyg.mov)

**Comment by madebysid** [comment]
Hey @emolr! I was seeing this too, but I pushed out a fix on the latest alpha (2.0.0-alpha.1016 as of writing). Could you try that version out and let me know if you're still seeing this?

For sake of clarity, the issue was that scalars that are part of relations are considered read-only (we should probably make this explicit in documentation). When you tried creating a record, Studio would populate these readonly values with "default" values. As a result, the generated Prisma Client query under the hood would be incorrect, and the query engine binary would reject the `update` request, causing the crash.
So as a side effect, Studio now supports readonly fields and shows these to you in the table if you try to edit them. Let me know how you think it feels too :D

**Comment by seanemmer** [comment]
I can confirm this is now fixed!
